staging/kos/pkg/controllers/ipam/ipam.go

```diff
@@ -53,6 +54,10 @@ const (
 	owningAttachmentIdxName = "owningAttachment"
 	attachmentSubnetIdxName = "subnet"
 
+	creation = "creation"
```

Comment:
Perhaps a slightly more specifc names would be good, such as `opCreation`?

Reply:
Fixed in 8ac9883.